python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -722,10 +737,50 @@ async def configure(self, config):
         # butler data path.
         self.dataPath = config.dataPath
 
+        self.large_defocus = config.large_defocus
+
+        self.threshold = config.threshold
+
+        self.max_iter = config.max_iter
+
     def set_metadata(self, metadata):
         # It takes about 300s to run the cwfs code, plus the two exposures
         metadata.duration = 300.0 + 2.0 * self.exposure_time
         metadata.filter = f"{self.filter},{self.grating}"
 
     async def run(self):
-        pass
+
+        total_focus_offset = 0.0
+        for i in range(self.max_iter):
+
+            self.log.debug(f"CWFS iteration {i+1} starting...")
+
+            self.intra_visit_id = None
```

Comment:
maybe just add a comment saying "Setting visit_id's to None so new data is taken"